crates/prelude/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Document all public API
 - Remove deprecated `println!` macro
 - Add `Sha384` and `HmacSha384` to the `rust-crypto` feature
+- Led returns error instead of trapping when indexing out of bounds.
```

Comment:
nit: the changelog is "append to top" and ideally starts with a verb describing the change, for example

```
- Change `led::{get,set}()` to never trap and return an error instead
```